code/demo/android-native/app/src/main/AndroidManifest.xml

```diff
@@ -10,7 +11,7 @@
 
         <!-- Our activity is the built-in NativeActivity framework class.
              This will take care of integrating with our NDK code. -->
-        <activity android:name=".MainActivity"
+        <activity android:name="android.app.NativeActivity"
```

Comment:
why did you drop the MainActivity?
It added Immersive Mode

Reply:
For a native activity, "immersive mode" should (as far as I know) be set in native code. From what I can see, the sample app runs in perfect full-screen mode, so I would assume that the "immersive mode" is indeed correctly initialized.

Anyway, the original `MainActivity` _incorrectly_ loaded the native lib, indicating a copy/paste error from non-native variant.

Reply:
Ok, I think I get it now. On older devices you have those overlay buttons. In that case, those `immersive` hacks are indeed needed. Will revert this part.